### DISCUSSION TITLE
What't the different of "gltfio-v1.12.2-lite-android.aar"  and "gltfio-v1.12.2-android.aar"

### DISCUSSION THREAD

**Discussion by dugufly** [google/filament#4594] [Q&A]
What't the different of the two version: "gltfio-v1.12.2-lite-android.aar"  and "gltfio-v1.12.2-android.aar"

**Comment by romainguy** [comment]
gltfio "lite" supports fewer advanced glTF features (extensions mostly). If binary size isn't an issue use the "full" version.

**Reply by StevenMOG** [reply]
@romainguy Is there some documentation / code file where I can see which features/extensions are not supported in the lite version?
